terratorch/models/__init__.py

```diff
@@ -6,6 +6,10 @@
 from terratorch.models.smp_model_factory import SMPModelFactory
 from terratorch.models.timm_model_factory import TimmModelFactory
 from terratorch.models.generic_unet_model_factory import GenericUnetModelFactory
+try:
+    from terratorch.models.wxc_model_factory import WxCModelFactory
+except:
+    print("granitewcx is not installed.")
```

Comment:
It is circumventing the major issue. These granitewxc and prithviwxc don't work for python <3.11. 
I'd remove as soon as they solve this issue. 
I don't want to import it, since it is not used during these tests. It' is just a way to run without errors.